src/components/TableBody.js

```diff
@@ -11,12 +14,21 @@ class TableBody extends Component {
     var lockstate = this.props.lockstate || 'unlocked';
     var total_working_hours = this.props.totaldays * 8;
     var total = 0;
+    // clear weekends from data
+    var weekday = [];
+    var i = 0;
+    Object.values(data).forEach(value => {
+      if (moment(value.event_date).isoWeekday() < 6) {
+      } else {
+        weekday.push(i)
+      }
+      i++;
+    });
+    for (var idx = weekday.length -1; idx >= 0; idx--)
```

Comment:
Just for curiosity: What does `idx` mean?

It looks like a loop that goes from a number down to 0?

Reply:
yes just index.. my own variable